### DISCUSSION TITLE
Widget inclusion in Bring to top Hotkey.

### DISCUSSION THREAD

**Discussion by debakl3** [baffler/Transparent-Twitch-Chat-Overlay#276] [Ideas]
I love this little package, I'm a laptop gamer and prefer to use a single monitor for performance reasons.

The game I play has an "exclusive desktop mode" that is not a true full desktop mode.

So I managed to get TTO running in exclusive desktop mode, then I press my bring to top hotkey, click a few times anywhere in the game and the the overlay stays until I alt tab, and I just repeat the process.

I recently added the streamlabs widget to show me follows/subs/raids etc. as the kapchat/jchat TTO overlay were hiding moobot from displaying this information in chat. I somehow got it working similar to how normal TTO does, I thought it may have been the allow multiple instances option, but maybe I have to tick and untick that every time I launch the game.

My idea is to port over the hotkey configurations to the widget, or perhaps share any CSS you might have used on the native TTO that we can then copy over to the widget to use.

FYI both the TTO kapchat and Streamlabs widget work perfectly in borderless fullscreen (their intended functionality), I'm just a glutton for a few more frames/responsiveness.

**Comment by baffler** [comment]
What you are asking for is that the widgets be brought to the top when you press the keybind for "Bring To Top Hotkey"? I think I did overlook that, it is setup to only work with the main window but I can add in the feature for it to work with the widgets as well.

**Reply by debakl3** [reply]
Fantastic, I appreciate the time you put in, to this project.
